.github/workflows/release.yml

```
@@ -303,25 +304,28 @@ jobs:
       - name: Download revive-wasm
         uses: actions/download-artifact@v4
         with:
-          name: revive-wasm
+          name: resolc-wasm.tar.gz
           path: resolc-wasm/
 
       - name: Download revive-linux
         uses: actions/download-artifact@v4
         with:
-          name: revive-linux
+          name: resolc-static-linux.tar.gz
           path: resolc-linux/
 
       - name: Download revive-macos
         uses: actions/download-artifact@v4
         with:
-          name: revive-macos
+          name: resolc-macos.tar.gz
           path: resolc-macos/
 
       - name: create-release
         uses: softprops/action-gh-release@v2
         with:
-          body: ${{ needs.tag.outputs.RELEASE_NOTES }}
+          body: |
+            ${{ needs.tag.outputs.RELEASE_NOTES }}
+
+            _Note: the MacOS binary is unsigned and it needs to be made runnable using `xattr -c resolc-macos`_
```

Without the new heading it looks like it belongs to "Fixes".

```suggestion
            # Note for macOS Users
            The macOS binary is unsigned and it needs to be made runnable using `xattr -c resolc-macos`.
```